steam_wrapper/steam_wrapper.py

```diff
@@ -196,13 +197,13 @@ def do_cleanup(self):
         assert self.need_cleanup
         self.log.info(f"Deleting {self.relocated_source}")
         shutil.rmtree(self.relocated_source)
-        return [self.relocated_source]
+        return False
```

Comment:
I'd rather avoid static return values. Just don't check it in `apply()` if it doesn't vary.

Reply:
I'm treating it as integer, every branch must return boolean.

Reply:
The return value seemed not to be used anywhere. Now the return value is considered "should_restart"  boolean everywhere.

Reply:
I got that. Just suggesting to to return `self.two_steps` value from `apply()` method after migration is applied, and `False` otherwise, and not change other methods. IMO it would be easier to understand what's going on this way.
Like this:
```python
    def apply(self):
        if self.need_migration:
            self.do_migrate()
            return self.two_steps
        elif self.need_cleanup:
            self.do_cleanup()
        return False
```